### PR TITLE
Pin Python minor.minor version in tests

### PR DESCRIPTION
**Pull request by dstansby**
Might be a temporary workaround for https://github.com/brainglobe/cellfinder/issues/207

### CODE REVIEW

**Comment by adamltyson** [comment]
Just out of curiosity, is this just to get around the multiprocessing-logging error?

**Comment by dstansby** [comment]
Yes, this is just to get the tests working for now whilst the multiprocessing issue is sorted

**Comment by dstansby** [comment]
Oh, I also realised that the `coredev` (ie. with development version of `cellfinder-core`) were always being run, which wasn't what I intended as it doubles test times, so I'm going to try and fix that here too.

**Comment by adamltyson** [comment]
@dstansby is there any reason to not merge this? It's still marked as draft.

**Comment by dstansby** [comment]
Don't think there's a reason not to merge, I've taken it out of draft 👍

File: tox.ini [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{37,38,39}-{,coredev}
+envlist = py{37,38,39}{-corerelease,-coredev}
```
Comment (adamltyson) [inline tox.ini:3]: I'm not familiar with this syntax. `coredev` seems to defined below as the GH repo. Is `corerelease` a standard thing, or does it need to be defined?

**Comment by adamltyson** [comment]
Closing as (partially) resolved - https://github.com/brainglobe/cellfinder/issues/207#issuecomment-1342961568

**Pull request closed without merging**